src/components/designer/AutoBalanceButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Button } from 'antd';
+import { useStoreActions } from 'store';
+import { Network } from 'types';
+
+const Styled = {
+  Button: styled(Button)`
+    margin-left: 8px;
+  `,
+};
+
+interface Props {
+  network: Network;
+}
+
+const AutoBalanceButton: React.FC<Props> = ({ network }) => {
+  const { autoBalanceChannels } = useStoreActions(s => s.network);
+
+  const handleClick = async () => autoBalanceChannels({ id: network.id });
```

Comment:
I would suggest wrapping this call in a try/catch and displaying a notification after the rebalance completes so the user knows it succeeded or failed.